### PR TITLE
Added machine learning client using flask

### PR DESCRIPTION
**Pull request by krishrustagi**

### CODE REVIEW

File: emotunes-admin/src/main/java/com/emotunes/emotunes/service/impl/AdminServiceImpl.java [hunk 1]
```diff
@@ -194,24 +197,22 @@ private String getTitle(Tag tag) {
...
-        Emotion songEmotion = null;
-        try {
-            songEmotion = userSongModelService.predictEmotion(trainingModelId, songUrl);
-        } catch (IOException e) {
-            log.error("Error while adding user song mapping for user {} and song {}", userId, songId, e);
-            songEmotion = Emotion.NEUTRAL;
-        }
+            String userId, String songId, String songUrl, String modelWeightsUrl) {
+
+        LinkedMultiValueMap<String, String> map = new LinkedMultiValueMap<>();
```
Comment (SarthakSavasil) [inline emotunes-admin/src/main/java/com/emotunes/emotunes/service/impl/AdminServiceImpl.java]: How this is being used in flask server, aur iska naam map nhii hona chaiye

File: emotunes-server/pom.xml [hunk 2]
```diff
@@ -38,6 +38,14 @@
             <version>0.0.1-SNAPSHOT</version>
             <scope>compile</scope>
         </dependency>
+
+
```
Comment (krishrustagi) [inline emotunes-server/pom.xml]:
```suggestion
```

File: emotunes-admin/src/main/java/com/emotunes/emotunes/service/impl/AdminServiceImpl.java [hunk 3]
```diff
@@ -74,10 +76,12 @@ public String addSongs(List<MultipartFile> songFiles) { // todo: use multithread
     @Override
     public String registerUser(UserDto userDto) {
         if (Objects.isNull(userDao.findByEmailId(userDto.getEmailId()))) {
-            String trainingModelId = userDao.saveAndGetModelId(userDto);
+            String defaultModelWeightsUrl = "default"; // todo: update to default
+            userDto.setModelWeightsUrl(defaultModelWeightsUrl);
+            userDao.save(userDto);
             // todo: add model of user
             // todo: use kafka
```
Comment (SarthakSavasil) [inline emotunes-admin/src/main/java/com/emotunes/emotunes/service/impl/AdminServiceImpl.java]: we only have one pod, kafka ki jarurat nhi hai can remove todo or add to use multi threading

File: emotunes-server/pom.xml [hunk 4]
```diff
@@ -38,6 +38,13 @@
             <version>0.0.1-SNAPSHOT</version>
             <scope>compile</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.emotunes</groupId>
+            <artifactId>emotunes-feign</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
+            <scope>compile</scope>
```
Comment (SarthakSavasil) [inline emotunes-server/pom.xml:46]: default scope is compile time only

**Review by SarthakSavasil — APPROVED** [review]

File: emotunes-admin/src/main/java/com/emotunes/emotunes/service/impl/AdminServiceImpl.java (hunk 3 above)
Comment (krishrustagi) [reply emotunes-admin/src/main/java/com/emotunes/emotunes/service/impl/AdminServiceImpl.java]: haa

**Pull request merged**